behave_django/management/commands/behave.py

```diff
@@ -69,6 +72,18 @@ def add_behave_arguments(parser):  # noqa
         '--simple',
     ]
 
+    def validate_runner(value):
+        try:
+            return import_string(value)
+        except ImportError:
+            raise ArgumentTypeError("No module named '%s' was found." % value)
+
+    parser.add_argument(
+        '--behave-runner', action='store', dest='runner_class',
+        default='behave.runner.Runner', type=validate_runner,
```

Comment:
There you go! `validate_runner` is actually [a type cast](https://docs.python.org/3.6/library/argparse.html#type). A function name that conveys this responsibility would be nice. :+1:

Reply:
Not sure what you had in mind, added `_path`, waiting on your guidance.